src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/TaskbarJumpList.cs

```diff
@@ -12,83 +12,87 @@ internal static class TaskbarJumpList
     {
         internal static void CreateElevatedEntry(string title)
         {
-            // check if the current shell owns a window
-            GetStartupInfo(out StartUpInfo startupInfo);
-            var STARTF_TITLEISLINKNAME = 0x00000800;
-            if (startupInfo.lpTitle == null || (startupInfo.dwFlags & STARTF_TITLEISLINKNAME) != STARTF_TITLEISLINKNAME)
+            if (Environment.UserInteractive)
```

Comment:
Lol. I will remove it then. The idea behind this was to not even have a single COM call in non-interactive mode (but it will should stop after the call to getstartupinfo then)

Reply:
I think it should be PowerShell internal flag - whether it interacts with an user.